cluster/aws/util.sh

```diff
@@ -352,6 +352,34 @@ function wait-for-instance-running {
   done
 }
 
+function allocate-elastic-ip {
+  $AWS_CMD allocate-address --domain vpc --output text | cut -f3
```

Comment:
There's a pretty strict bash style (as you've probably noticed!)  I think in this case, we can't quote it, but curly brackets help us get closer to full-adherence to the "style guide".  (I'm not sure if the style guide is actually written down anywhere, but we set options to exit on error, and require variable declaration; also I find it helpful to copy an example from elsewhere in the script when testing e.g. whether a variable is empty)